console/account/how-to/close-account.mdx

```diff
@@ -1,69 +1,50 @@
 ---
 meta:
   title: How to close an account
-  description: This page explains the different methods to close an account and how to reactivate it.
+  description: This page explains how to close your account and how to reopen it.
 content:
   h1: How to close an account
-  paragraph: This page explains the different methods to close an account and how to reactivate it.
+  paragraph: This page explains how to close your account and how to reopen it.
 tags: account close cancel delete deactivate close-account
 dates:
-  validation: 2023-11-02
+  validation: 2024-03-02
   posted: 2021-06-22
 categories:
   - console
 ---
 
-You can close your account using two methods:
+If you do not want to use Scaleway services anymore, you can close your account and/or [erase your personal data](/console/account/how-to/manage-my-personal-data-gdpr/#how-to-erase-your-account-and-personal-data). 
 
-- by deactivating your Organization, which allows you to [reactivate it](#how-to-reactivate-an-organization) at a later date
-- by permanently deleting your account and personal data.
+## Closing your account or erasing your personal data: what’s the difference
 
-<Macro id="requirements" />
+<Macro id="close-vs-erase" />
 
-- A Scaleway account logged into the [console](https://console.scaleway.com)
-- [Ownership](/identity-and-access-management/iam/concepts/#owner) of the Organization you intend to delete
+## How to close your account
 
-## How to deactivate your Organization
+Closing your account will permanently delete your Organization’s servers, resources, services, and backups. However, you keep access to the Scaleway console. 
 
-Deactivating an Organization will permanently delete your Organization's servers, resources, services, and backups. It will also deactivate the account. However, you can choose to reactivate your account at a later date, eliminating the need to recreate an account from scratch.
+<Macro id="requirements" />
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Ownership](/identity-and-access-management/iam/concepts/#owner) of the Organization you intend to delete
 
 <Message type="note">
-  There is a delay of up to 10 days between the time your Organization is closed and the time your resources are effectively deleted. We recommend you to manually delete all your resources before you close your Organization. 
+  We recommend you delete all resources before closing your account. Otherwise, the remaining resources will be billed for usage before their manual deletion (this can take up to 10 days).
```

Comment:
```suggestion
  We recommend that you delete all resources before closing your account. Otherwise, the remaining resources will be billed for usage before their manual deletion, which can take up to 10 days.
```